modules/core/src/testFixtures/java/org/apache/ignite/internal/testframework/matchers/CompletableFutureMatcher.java

```diff
@@ -145,7 +146,7 @@ public static <T> CompletableFutureMatcher<T> willBe(Matcher<T> matcher) {
      * @param <T> value type
      * @return matcher
      */
-    public static <T> CompletableFutureMatcher<T> willBe(T value) {
+    public static <T> CompletableFutureMatcher<T> willBe(@Nullable T value) {
```

Comment:
willBe(null) should never be used; instead, willBe(nullValue()) is preferred, so this annotation is not needed here. Please don't replace willBe(null) in the code, I've already done this in my PR that will soon be merged

Reply:
fix it